runtime/src/main/scala-3/mdoc/internal/document/Printing.scala

```diff
@@ -3,14 +3,18 @@ package mdoc.internal.document
 import Compat.TPrint
 
 object Printing {
-    inline def stringValue[T](value: T) = value.toString
+    inline def stringValue[T](value: T) = nullableToString(value)
     inline def typeString[T](tprint: TPrint[T]) = tprint.render
  
     inline def print[T](value: T, out : StringBuilder, width: Int, height: Int) = {
-       out.append(value.toString)
+       out.append(nullableToString(value))
     }
 
     inline def printOneLine[T](value: T, out : StringBuilder, width: Int) = {
-       out.append(value.toString.replace("\n",""))
+       out.append(nullableToString(value).replace("\n",""))
+    }
+
+    inline private def nullableToString[T](value: T) = {
```

Comment:
I believe this can be replaced with String.valueOf or an interpolated string.

Reply:
I think that would be equivalent. I can change it later on when doing some work around it. Or do you think this might work more correctly?